app/helpers/star_helper.rb

```diff
@@ -15,6 +19,8 @@ def star_rating(rating)
       stars << '<i class="fas fa-star-half-alt" aria-hidden=”true”></i>'
     end
 
+    stars << " (#{rating})" if (rating_decimal != 0.0 && rating_decimal != 0.5)
```

Comment:
As discussed, comment here would be helpful

```suggestion
    # Only multiples of 0.5 can be shown in visual stars, so append the numeric rating if the rating is not a multiple of 0.5 (number of stars shown will be rounded down to nearest multiple)
    stars << " (#{rating})" if (rating_decimal != 0.0 && rating_decimal != 0.5)
```